src/win/tty.c

```diff
@@ -1636,6 +1667,51 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
     uv_buf_t buf = bufs[i];
     unsigned int j;
 
+    if (uv__vterm_state == UV_SUPPORTED && buf.len > 0) {
+      utf16_buf_used = MultiByteToWideChar(CP_UTF8,
+                                           0,
+                                           buf.base,
+                                           buf.len,
+                                           NULL,
+                                           0);
+
+      if (utf16_buf_used == 0) {
+        *error = GetLastError();
+        break;
+      }
+
+      max_len = (utf16_buf_used + 1) * sizeof(WCHAR);
+      if (max_len > MAX_CONSOLE_CHAR) {
+        utf16_ansi_buf = (WCHAR*)malloc(max_len);
+        if (!MultiByteToWideChar(CP_UTF8,
```

Comment:
I think this section can be rewritten so we only have one `MultiByteToWideChar` call